Ome/JavaScript/6_Assignment/JavaScriptAssignment6_Q5.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous" />
+    <!-- Option 1: Bootstrap Bundle with Popper -->
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/js/bootstrap.bundle.min.js"
+        integrity="sha384-gtEjrD/SeCtmISkJkNUaaKMoLD0//ElJ19smozuHV6z3Iehds+3Ulb9Bn9Plx0x4"
+        crossorigin="anonymous"></script>
+
+    <title>Factorial for each number</title>
+
+    <style>
+        .container {
+            margin-top: 15%;
+        }
+
+        input {
+            margin: 5px;
+        }
+    </style>
+</head>
+
+<body class="container">
+    <div class="container-box, shadow p-3 mb-5 bg-body rounded">
+        <h4>Check factorial for each number :</h4>
+
+        <br /><br />
+        <button type="submit" onclick="SumOfFactEachArray()" class="btn btn-primary">
+            Click Here
+        </button>
+    </div>
+
+    <script>
+        function SumOfFactEachArray() {
+            let FactNumber = [];
+
+            while ("Sum of Factorial for each number") {
+                let n = prompt("Enter the number");
+                if (n == "X" || n == null || n == "x") {
+                    break;
+                }
+                FactNumber.unshift(parseInt(n));
+            }
+
+            let factorial = [];
+            for (let i = 0; i < FactNumber.length; i++) {
+                let f = fact(FactNumber[i]);
+                factorial.unshift(f);
+            }
+
+            function fact(n) {
+                let f = n;
+                for (let i = 1; i < n; i++) f = f * i;
+                return f;
+            }
+            document.write(`Factorial in descending Order :${factorial}`);
```

Comment:
factorial must be printed in descending order,use reverse method

Reply:
Like this.
```suggestion
            document.write(`Factorial in descending Order :${factorial.sort().reverse();}`);
```

Reply:
Thank you so much Qadir i ll implement this.

Reply:
Welcome